crates/superposition_types/src/api/experiments.rs

```diff
@@ -241,6 +241,7 @@ pub struct ExperimentListFilters {
     pub created_by: Option<CommaSeparatedStringQParams>,
     pub sort_on: Option<ExperimentSortOn>,
     pub sort_by: Option<SortBy>,
+    pub global_experiments_only: Option<bool>,
```

Comment:
Instead of Option<bool> can we do:
```suggestion
    #[serde(default = false)]
    pub global_experiments_only: bool,
```

Reply:
why this ?
we keep all the filters as optional only no

quite a few logic in frontend is based on that, few behaviors would break with this

Reply:
yes, default behaviors would break on the pages